draft-ietf-core-groupcomm-bis.md

```diff
@@ -611,6 +612,19 @@ As part of group maintenance operations (see {{sec-group-maintenance}}), additio
 
 The key management operations mentioned above are entrusted to the Group Manager responsible for the OSCORE group {{I-D.ietf-core-oscore-groupcomm}}, and it is RECOMMENDED to perform them according to the approach described in {{I-D.ietf-ace-key-groupcomm-oscore}}.
 
+## Proxy Security # {#chap-proxy-security}
+
+Different solutions may be selected for secure group communication via a proxy depending on proxy type, use case and deployment requirements. In this section the options based on Group OSCORE are listed.
+
+For a client performing a group communication request via a forward-proxy, end-to-end security should be implemented. The client then creates a group request secured by Group OSCORE and unicasts this to the proxy. The proxy adapts the request from a forward-proxy request to a regular request and multicasts this adapted request to the indicated CoAP group. During the adaptation, the OSCORE security i.e. COSE signature on the request remains valid. The first leg of communication from client to proxy can optionally be protected with an additional protocol such as (D)TLS or unicast OSCORE.
```

Comment:
Proposed rephrasing:

For a client performing a group communication request via a forward-proxy, end-to-end security should be implemented. The client then creates a group request protected with Group OSCORE and unicasts this to the proxy. The proxy adapts the request from a forward-proxy request to a regular request and multicasts this adapted request to the indicated CoAP group. During the adaptation, the security provided by Group OSCORE persists, both when using the group mode or the pairwise mode. The first leg of communication between client and proxy can optionally be further protected, e.g., by using (D)TLS and/or OSCORE.